### PR TITLE
Added recipe steps for user and email settings

### PR DESCRIPTION
**Pull request by jersiovic**
I needed to set for my tests some of these settings, so in order to automate the job I needed those steps, hope it is ok for tu to merge it

### CODE REVIEW

File: src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPasswordConfirmation.cshtml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿@{
+    ViewLayout = "Layout__Login";
+}
+
+<h1>@T["Reset Password"]</h1>
+<p>
+    @T["Your password has been reset. Melon"]
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPasswordConfirmation.cshtml:7]: Melon?

File: src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPassword.cshtml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="form-group">
+        <label asp-for="NewPassword" class="col-md-3 control-label">@T["New password"]</label>
+        <div class="col-md-9">
+            <input asp-for="NewPassword" class="form-control" />
+            <div id="passwordStrength"></div>
+            <span asp-validation-for="NewPassword" class="text-danger"></span>
+        </div>
+    </div>
+    <div class="form-group">
+        <label asp-for="PasswordConfirmation" class="col-md-3 control-label">@T["New password confirmation Melon"]</label>
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPassword.cshtml:27]: Melon?

File: src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ForgotPasswordConfirmation.cshtml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿@{
+    ViewLayout = "Layout__Login";
+}
+
+<h1>@T["Forgot Password confirmation Melon"]</h1>
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ForgotPasswordConfirmation.cshtml:5]: Melon?

File: src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ForgotPassword.cshtml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+<h2>@T["Forgot password?"]</h2>
+<h4>@T["Please check your email or user name to reset your password."]</h4>
+<hr />
+<div class="row">
+    <div class="col-md-8">
+        <form asp-controller="ResetPassword" asp-action="ForgotPassword" method="post" class="form-horizontal">
+            <div asp-validation-summary="All" class="text-danger"></div>
+            <div class="form-group">
+                <label asp-for="UserIdentifier" class="col-md-4 control-label">@T["Email or user name Melon"]</label>
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ForgotPassword.cshtml:15]: Melon?

**Comment by agriffard** [comment]
See also this PR #5556 and this issue #5558

**Comment by agriffard** [comment]
Build error:
`Recipes\SmtpSettingsStep.cs(6,19): error CS0234: The type or namespace name 'Recipes' does not exist in the namespace 'OrchardCore' (are you missing an assembly reference?) `

**Comment by agriffard** [comment]
The SMTP step needs is a bit sensitive because it can contain password information:
https://github.com/OrchardCMS/OrchardCore/pull/5556#issuecomment-589302841

File: src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPasswordConfirmation.cshtml (hunk 1 above)
Comment (jersiovic) [reply src/OrchardCore.Themes/TheTheme/Views/OrchardCore.Users/ResetPassword/ResetPasswordConfirmation.cshtml:7]: upps sorry this is a test I made cause I was not able to override this cshtml I forgot to undo those changes

**Comment by jersiovic** [comment]
the build error was fixed on next force-push of the branch.
Ok I will think about those points you commented. Thank you

**Comment by JoshLefebvre** [comment]
Any further progress on this PR? This is an important one for me as I would really like to automate the process of setting up Smtp settings for new tenants

**Comment by deanmarcussen** [comment]
Take a look at this concept and see if it meets your needs https://github.com/OrchardCMS/OrchardCore/pull/6421

**Comment by JoshLefebvre** [comment]
@deanmarcussen Wow great timing! Yep the encrypted option would work out perfectly for my scenario. Thanks!

**Comment by jersiovic** [comment]
I close this PR, cause what deanmarcussen is better

**Pull request closed without merging**